src/modules/material-requests/dto/material-requests.dto.ts

```diff
@@ -81,3 +103,57 @@ export class MaterialRequestsQueryDto {
 	})
 	orderDirection? = SortDirection.desc;
 }
+
+export class CreateMaterialRequestDto {
+	@IsString()
+	@ApiProperty({
+		type: String,
+		description: 'The object schema identifier',
+		example: '9f2479c1-4489-4bd0-86b3-881b9449a8c0',
+	})
+	object_id: string;
```

Comment:
Everything to the client side should be camel case
```suggestion
	objectId: string;
```